lib/rom/setup/auto_registration.rb

```diff
@@ -10,7 +10,7 @@ class AutoRegistration
 
     include Options
 
-    option :namespace, reader: true, type: [TrueClass, FalseClass], default: true
+    option :namespace, reader: true, type: [TrueClass, FalseClass, String], default: true
```

Comment:
Oh wait of you're right. Sorry, this is what happens when I read code at midnight on a Wednesday 😆